streamlit_drawable_canvas/frontend/src/lib/rect.ts

```diff
@@ -22,6 +23,7 @@ class RectTool extends FabricTool {
     this.strokeWidth = strokeWidth
     this.strokeColor = strokeColor
     this.fillColor = fillColor
+    this.minSize = strokeWidth
```

Comment:
Same as above

I don't think this is going to be used outside the class right? We can make it private if that's the case, by prepending with an underscore, so `this._minsize` and `_minsize: number` in the declaration

If we make this private, let's rename it more explicitly to `minLength`